packages/obonode/obojobo-chunks-list/editor-component.js

```diff
@@ -54,10 +54,11 @@ const List = props => {
 			/>
 		)
 	}
+	console.log(props);
 
 	return (
 		<Node {...props}>
-			<div className={'text-chunk obojobo-draft--chunks--list pad'}>
+			<div className={`text-chunk obojobo-draft--chunks--list pad li-spacing-${props.element.content.spacing}`}>
```

Comment:
Naming convention we're using for classes that are added dynamically via the react component are prefix them with `is-`, so I'm thinking instead of `li-spacing-*` we want `is-spacing-*` (i.e. `is-spacing-moderate`, etc)